L1Trigger/L1THGCal/python/hgcalVFEProducer_cfi.py

```diff
@@ -72,6 +81,7 @@
         eeDigis = cms.InputTag('simHGCalUnsuppressedDigis:EE'),
         fhDigis = cms.InputTag('simHGCalUnsuppressedDigis:HEfront'),
         bhDigis = cms.InputTag('simHGCalUnsuppressedDigis:HEback'),
+        noseDigis = cms.InputTag('simHFNoseUnsuppressedDigis:HFNose'),
```

Comment:
What happens if we run on a sample that doesn't contain the nose digis?

Reply:
tested successfully with runtheMatrix on scenario

22034.0_TTbar_14TeV+TTbar_14TeV_TuneCUETP8M1_2026D46_GenSimHLBeamSpotFull14+DigiFullTrigger_2026D46+RecoFullGlobal_2026D46+HARVESTFullGlobal_2026D46